internal/core/regularsurface.cpp

```diff
@@ -95,24 +95,30 @@ Point RegularSurface::from_cdp(
     return this->m_plane.m_inverse_transformation * point;
 }
 
-float RegularSurface::value(
-    std::size_t const row,
-    std::size_t const col
-) const noexcept (false) {
-    if (row >= this->nrows()) throw std::runtime_error("Row out of range");
-    if (col >= this->ncols()) throw std::runtime_error("Col out of range");
 
-    return this->m_data[row * this->ncols() + col];
-};
+std::pair<std::size_t,std::size_t> RegularSurface::to_pair(std::size_t row, std::size_t col) {
+    return std::pair<std::size_t,std::size_t>(row, col);
+}
 
-float RegularSurface::value(std::size_t i) const noexcept (false) {
-    if (i >= this->size()) throw std::runtime_error("index out of range");
+float& RegularSurface::operator[]( std::size_t i ){
+    if (i < 0 || i >= this->size()) throw std::runtime_error("Index out of range");
+    return this->m_data[i];
+};
```

Comment:
I was unaware of the distinction between definitions and declarations when it comes to semicolon. 

For educational purpose, what is the difference [here](https://github.com/equinor/vds-slice/blob/d0d19291f0e47086463aa6678752f78c3a8fe12c/internal/core/regularsurface.cpp#L21) and [here](https://github.com/equinor/vds-slice/blob/d0d19291f0e47086463aa6678752f78c3a8fe12c/internal/core/regularsurface.cpp#L11).

Reply:
:sweat_smile: 
None.
Didn't know that we put semicolon after some functions  :smile: 
So apparently we already have a mix of style in the code, so no need to remove those seimcolons unless you want to :slightly_smiling_face: 

I think it might happen after we copy the declaration into .cpp file to make a definition out of it.